package.json

```diff
@@ -79,13 +79,13 @@
     "lint-staged": "11.1.2",
     "mocha": "9.1.1",
     "pg": "8.7.1",
-    "prettier": "2.3.2",
+    "prettier": "2.4.1",
```

Comment:
After updating prettier, we may need to run the formatter

Reply:
ran, nothing changes